winui/Avatar-view/visual-styles.md

```diff
@@ -1,10 +1,364 @@
+MediumCircleTextBlock.TextAlignment = TextAlignment.Center;
+MediumCircleTextBlock.FontSize = 10;
+
+SfAvatarView avatarView3 = new SfAvatarView();
+avatarView3.AvatarShape = AvatarShape.Circle;
+avatarView3.AvatarSize = AvatarSize.Medium;
+avatarView3.ContentType = AvatarContentType.AvatarCharacter;
+avatarView3.AvatarCharacter = AvatarCharacter.Avatar13;
+
+TextBlock SmallCircleTextBlock = new TextBlock();
+SmallCircleTextBlock.Text = "Small";
+SmallCircleTextBlock.FontSize = 12;
+SmallCircleTextBlock.TextAlignment = TextAlignment.Center;
+SmallCircleTextBlock.FontSize = 10;
+
+SfAvatarView avatarView4 = new SfAvatarView();
+avatarView4.AvatarShape = AvatarShape.Circle;
+avatarView4.AvatarSize = AvatarSize.Small;
+avatarView4.ContentType = AvatarContentType.AvatarCharacter;
+avatarView4.AvatarCharacter = AvatarCharacter.Avatar13;
+
+TextBlock XSmallCircleTextBlock = new TextBlock();
+XSmallCircleTextBlock.Text = "Extra-small";
+XSmallCircleTextBlock.FontSize = 12;
+XSmallCircleTextBlock.TextAlignment = TextAlignment.Center;
+XSmallCircleTextBlock.FontSize = 10;
+
+SfAvatarView avatarView5 = new SfAvatarView();
+avatarView5.AvatarShape = AvatarShape.Circle;
+avatarView5.AvatarCharacter = AvatarCharacter.Avatar13;
+avatarView5.AvatarSize = AvatarSize.ExtraSmall;
+avatarView5.ContentType = AvatarContentType.AvatarCharacter;
+
+mainGrid.Children.Add(XLargeCircleTextBlock);
+mainGrid.Children.Add(avatarView1);
+mainGrid.Children.Add(LargeCircleTextBlock);
+mainGrid.Children.Add(avatarView2);
+mainGrid.Children.Add(MediumCircleTextBlock);
+mainGrid.Children.Add(avatarView3);
+mainGrid.Children.Add(SmallCircleTextBlock);
+mainGrid.Children.Add(avatarView4);
+mainGrid.Children.Add(XSmallCircleTextBlock);
+mainGrid.Children.Add(avatarView5);
+
+Grid.SetRow(avatarView5, 0);
+Grid.SetRow(avatarView4, 0);
+Grid.SetRow(avatarView3, 0);
+Grid.SetRow(avatarView2, 0);
+Grid.SetRow(avatarView1, 0);
+
+Grid.SetColumn(avatarView5, 0);
+Grid.SetColumn(avatarView4, 1);
+Grid.SetColumn(avatarView3, 2);
+Grid.SetColumn(avatarView2, 3);
+Grid.SetColumn(avatarView1, 4);
+
+Grid.SetRow(XSmallCircleTextBlock, 1);
+Grid.SetRow(SmallCircleTextBlock, 1);
+Grid.SetRow(MediumCircleTextBlock, 1);
+Grid.SetRow(LargeCircleTextBlock, 1);
+Grid.SetRow(XLargeCircleTextBlock, 1);
+
+Grid.SetColumn(XSmallCircleTextBlock, 0);
+Grid.SetColumn(SmallCircleTextBlock, 1);
+Grid.SetColumn(MediumCircleTextBlock, 2);
+Grid.SetColumn(LargeCircleTextBlock, 3);
+Grid.SetColumn(XLargeCircleTextBlock, 4);
+stack.Children.Add(mainGrid);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with circle visual styles](avatarview_images/winui_circle_style_avatarview.png)
+
+## Square
+
+For the square style, you can use the `AvatarSize` property to set the avatar's pre-defined square size, which includes the following values:
+
+* Extra-large
+* Large
+* Medium
+* Small
+* Extra-small
+
+The following code sample illustrates how to set the visual styles of AvatarView to `Square`.
```

Comment:
The following code sample illustrates how to set the visual styles of AvatarView to `Square.`